### ISSUE TITLE
Dual License?

### ISSUE THREAD

**Issue by icosahedron** [squid-box/SevenZipSharp#18]
Would you consider licensing under LGPL also?  GPL 3 is a bit more restrictive than my company is willing to allow.

**Comment by squid-box** [comment]
I'm not sure that is up to me. The original project from CodePlex was licensed under GPL 3, and as I understand this pretty much means it applies to it forever.

I haven't read to on it too much though, so if you know otherwise please explain it to me :)

**Comment by icosahedron** [comment]
That could be.  I'm not sure how copyright reassignments work.  Did the former owner reassign the project to you, or is this a simple fork?

**Comment by squid-box** [comment]
The former "owner" has said he will point to my fork, but the original author seems to have abandoned the project before CodePlex went down.

Guess I could see if I can track the original author down and ask him directly.

**Comment by squid-box** [comment]
It does look pretty dark:

```
Note that the LGPL is compatible with the GPL: you can decide to "upgrade" to GPL and incorporate it in a wholly GPL licensed project if you wish. You can't however go the other way and re-license GPL licensed code as LGPL.
```

From question 9 of https://resources.whitesourcesoftware.com/blog-whitesource/top-10-gpl-license-questions-answered

**Comment by icosahedron** [comment]
I agree with your assessment.  I'm going to just assume that the answer is no.  Thanks for your quick replies and thorough research.

**Comment by squid-box** [comment]
Actually it looks like I've misread from the start, the original project was released under LGPL 2.1, later halfway updated to LGPL 3.0 - I misread it as GPL 3.0.

I will correct this mistake now.
